src/js/components/Select/__tests__/Select-test.js

```diff
@@ -444,6 +444,7 @@ describe('Select', () => {
 
   test('select an option with keypress', () => {
     const onChange = jest.fn();
+    window.HTMLElement.prototype.scrollIntoView = function() {};
```

Comment:
Not sure why the scrollIntoView statements were added to some of the tests. I don't think these tests should change because of this PR